shared/src/commonMain/kotlin/com/bunbeauty/shared/data/repository/UserRepository.kt

```diff
@@ -82,21 +89,39 @@ class UserRepository(
         networkConnector.patchSettings(token, PatchUserServer(isActive = false))
     }
 
-    suspend fun saveProfileLocally(profile: ProfileServer?) {
+    private suspend fun saveProfileLocally(profile: ProfileServer?) {
         if (profile != null) {
             dataStoreRepo.saveUserUuid(profile.uuid)
             userDao.insertUser(profileMapper.toUserEntity(profile))
             userAddressDao.insertUserAddressList(profileMapper.toUserAddressEntityList(profile))
-            profile.orders.flatMap { orderServer ->
-                orderServer.oderProductList.map { orderProductServer ->
-                    orderMapper.toOrderWithProductEntity(orderServer, orderProductServer)
+            profile.orders.forEach { orderServer ->
```

Comment:
It's outdated logic. We shouldn't save orders from profile. This list is always empty